ui/app/styles/core/columns.scss

```diff
@@ -122,6 +122,11 @@
     width: 25%;
   }
 
+  &.is-4 {
```

Comment:
named `is-4` to keep with the other class names in this file, makes add and trash button the same width 
<img width="1057" alt="Screenshot 2023-11-14 at 12 35 51 PM" src="https://github.com/hashicorp/vault/assets/68122737/fc2d5eea-5cee-4710-9b6b-19486fc8aad1">